packages/medusa/src/utils/define-routes-config.ts

```diff
@@ -32,8 +32,16 @@ export function defineRoutesConfig<
       next: MedusaNextFunction
     ) => any)[]
   }
->(routes: Route[]): MiddlewaresConfig {
+>(
```

Comment:
Suggestion : what about doing something similar to the defineLink and adding a symbol to the returned object such that when we import the file in the loader we dont have to check default or not and just look for that symbol in Object.values(...)?

Let me know what you think

Reply:
But that means you can export the config via any named export. To be honest, feels a bit weak, since in this case the `default export` kind of acts as a Symbol + ensures consistency too.

Reply:
I am fine with either way. It was more a suggestion to not rely on default and more be able to let the user choose if he want to default export it or just export a named const for any reason (like carrying meaning)

Reply:
Maybe defineLink was another thing because the user is more likely to reimport and consume it and having a named export sounds better

Reply:
> let the user choose if he want to default export it or just export a named

Yeah, that's the thing. I am not sure if the freedom to choose the export names is something we need. If this becomes a conscious choice across the entire codebase, then yeah it might make sense to go with your suggestion. :)